doc/api/assert.md

```diff
@@ -8,410 +8,495 @@ used in application code via `require('assert')`. However, `assert` is not a
 testing framework, and is not intended to be used as a general purpose assertion
 library.
 
-The API for the `assert` module is [Locked][]. This means that there will be no
-additions or changes to any of the methods implemented and exposed by
-the module.
+The API for the `assert` module is [Locked]. This means that there will be no
+additions or changes to any of the methods implemented and exposed by this module.
 
 ## assert(value[, message])
 <!-- YAML
 added: v0.5.9
 -->
 
-An alias of [`assert.ok()`][] .
-
-```js
-const assert = require('assert');
-
-assert(true);  // OK
-assert(1);     // OK
-assert(false);
-  // throws "AssertionError: false == true"
-assert(0);
-  // throws "AssertionError: 0 == true"
-assert(false, 'it\'s false');
-  // throws "AssertionError: it's false"
-```
+An alias of [`assert.ok()`].
 
 ## assert.deepEqual(actual, expected[, message])
 <!-- YAML
 added: v0.1.21
 -->
 
-Tests for deep equality between the `actual` and `expected` parameters.
-Primitive values are compared with the equal comparison operator ( `==` ).
+* `actual` {mixed} A value to check
```

Comment:
I used `mixed` because that is what I'm used to seeing in the online PHP documentation. I figured other people coming from PHP might find that term more familiar than something else. \*shrug\*